pkg/fixtures/fixtures.go

```diff
@@ -274,22 +274,23 @@ func (fxt *Fixture) parseMap(params map[string]interface{}, parent string, index
 	return data
 }
 
-func (fxt *Fixture) parseArray(params []interface{}, parent string, index int) []string {
+func (fxt *Fixture) parseArray(params []interface{}, parent string) []string {
```

Comment:
I think a docstring for this function would help in the future (should have been already here).
Reading the function gets a bit confusing if someone is trying to figure out what shape can be expected for `params` and `fmt.Sprintf("%s[]=%s", parent, fxt.parseQuery(v.String())))` is pretty confusing to parse as a human.

Reply:
Good callout, I'll try to add some more documentation here!

Reply:
Updated, ptal @pepin-stripe

Reply:
That's much better! 

There is a typo (`elemenent`) and I think it'd be a useful to show an example of possible input since the way url params are passed (`"address[city]=New New York"`, `"tax_id_data[1][value]=value_1"`, etc) can be pretty confusing.

not a blocker tho!

Reply:
Agreed, added those examples. ptal @pepin-stripe

Reply:
looks great now thanks :)